pkg/api/cachedcsi.go

```diff
@@ -0,0 +1,203 @@
+package api
```

Comment:
Why do these methods on `Cached` exist in another file?

Reply:
I figured it was nice to separate them since its the CSI stuff vs our stuff, but if that's against convention I will merge them!